api/src/services/items.ts

```diff
@@ -774,7 +774,7 @@ export class ItemsService<Item extends AnyItem = AnyItem> implements AbstractSer
 						? ['items.update', `${this.collection}.items.update`]
 						: `${this.eventScope}.update`,
 				meta: {
-					payload,
+					payload: payloadAfterHooks,
```

Comment:
does it make sense to use `payloadWithPresets` here?

Reply:
Yeah, updated to `payloadWithPresets` so the emitted action event payload matches the actual updated value 👍